src/Microsoft.ML.Core/Data/RoleMappedSchema.cs

```diff
@@ -98,68 +112,99 @@ public sealed class RoleMappedSchema
         private const string NameString = "Name";
         private const string FeatureContributionsString = "FeatureContributions";
 
+        /// <summary>
+        /// Instances of this are the keys of a <see cref="RoleMappedSchema"/>. This class also some important commonly used
```

Comment:
> some  [](start = 95, length = 6)

this phrasing looks weird for me, but what do I know

Reply:
You know lots Ivan!

---
In reply to: [200436663](https://github.com/dotnet/machinelearning/pull/496#discussion_r200436663) [](ancestors = 200436663)